modules/realsense_t265.py

```diff
@@ -0,0 +1,100 @@
+import traceback
+import sys
+import time
+
+import pyrealsense2 as rs
+
+class rs_t265:
+    """
+    Initialise variables for T265 camera
+    """
+    def __init__(self):
+        # Private
+        self._pipe = None
+
+    """
+    with __enter__ method opens a connected to the T265 camera
+    """
+    def __enter__(self):
+        self.open_connection()
+
+    """
+    with __exit__ method closes the connection to the T265 camera
+    """
+    def __exit__(self, exception_type, exception_value, traceback):
+        if traceback:
+            print(traceback.tb_frame)
+
+        self.close_connection()
+
+    """
+    Open a connected to the T265 camera
+    """
+    def open_connection(self):
+        cfg = rs.config()
+        cfg.enable_stream(rs.stream.pose)
+
+        self._pipe = rs.pipeline()
+        self._pipe.start(cfg)
+        print('rs_t265:T265 Connection Open')
+
+    """
+    Close connected to the T265 camera
+    """
+    def close_connection(self):
+        if self._pipe is None:
+            return
+
+        self._pipe.stop()
+        self._pipe = None
+
+        print('rs_t265:T265 Connection Closed')
+
+    """
+    Retrieve a data from the T265 camera
+    """
+    def get_frame(self) -> tuple:
+        frames = self._pipe.wait_for_frames()
```

Comment:
Is this blocking? If this is blocking, dont we want it runnning in a background thread using the same queue system? Or are we just gonna pump it straight into Rabbit mq?

Reply:
It's blocking, was assuming this was just going to be a single thread producer